test/acceptance/pages.py

```diff
@@ -451,6 +451,11 @@ def grade_entry(self, question, column):
         Returns: the tuple of source and value information for the requested grade
 
         """
+        self.wait_for_element_visibility(
+            self._bounded_selector('.question--{} .answer .answer__source__value'.format(question + 1)),
+            "Grade entry was present",
+            2
+        )
```

Comment:
@cahrens I wanted to call out this particular change - I was getting intermittent failures on my acceptance test due to accessing this value before it was present, this `wait_for_element_visibility` resolves that.

Everything else in this commit is just addressing your comments from before.